### PR TITLE
audio restart injector limits

### PR DESCRIPTION
**Pull request by howard-stearns**
Extend the test for number of active injectors to apply to injector restarts.

### CODE REVIEW

**Comment by howard-stearns** [comment]
https://app.asana.com/0/search/92931214910284/92889183540689

**Comment by howard-stearns** [comment]
There's a restartLimit.js attached to the asana task for testing. (Oddly, github won't let me attach a .js file to a comment.)

File: libraries/audio/src/AudioInjectorManager.cpp [hunk 1]
```diff
@@ -129,6 +129,16 @@ void AudioInjectorManager::run() {
 
 static const int MAX_INJECTORS_PER_THREAD = 40; // calculated based on AudioInjector time to send frame, with sufficient padding
 
+bool AudioInjectorManager::wouldExceedLimits() { // Should be called inside of a lock.
+    if (_injectors.size() >= MAX_INJECTORS_PER_THREAD) {
+        qDebug() << "AudioInjectorManager::threadInjector could not thread AudioInjector - at max of"
+            << MAX_INJECTORS_PER_THREAD << "current audio injectors.";
+        return true;
+    }
+    qDebug() << "current injectors:" << _injectors.size();
```
Comment (birarda) [inline libraries/audio/src/AudioInjectorManager.cpp]: Do we need this? It will pollute debug.

Comment (howard-stearns) [inline libraries/audio/src/AudioInjectorManager.cpp]: Quite right. Removing.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5d9ee84e6364a053c05275009fea50c98b69c7e2 available at: https://gist.github.com/8685d09f00f7479cbdfb<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7168.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7168.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6a987391d2d0f46682e29c912b75256755b55695 available at: https://gist.github.com/8685d09f00f7479cbdfb<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7168.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7168.dmg

**Pull request merged**